.gitignore

```diff
@@ -7,3 +7,4 @@ php.ini
 phpunit.xml
 apigen.phar
 site
+.idea
```

Comment:
Let's remove this. IDE-specific ignores are best left in users' own `.gitignore` files.